core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -255,12 +261,21 @@ private CopycatClient createClient() {
 
   private static List<Address> getClusterAddresses(RaftJournalConfiguration conf) {
     return conf.getClusterAddresses().stream()
-        .map(addr -> new Address(addr.getHostName(), addr.getPort()))
+        .map(addr -> new Address(addr))
         .collect(Collectors.toList());
   }
 
   private static Address getLocalAddress(RaftJournalConfiguration conf) {
-    return new Address(conf.getLocalAddress().getHostName(), conf.getLocalAddress().getPort());
+    return new Address(conf.getLocalAddress());
```

Comment:
Need to do the same in `RaftJournalSystem#removeQuorumServer`.

Reply:
done